plugins/labhub.py

```diff
@@ -279,14 +279,18 @@ def difficulty_level(user, iss):
                 2. The user belongs to developers or maintainers team as well as
                    newcomers team.
                 False if
-                1. A newcomer asks for assignment to an issue that has no
+                1. A member asks for assignment to an issue that has
+                   initiatives/gci label.
+                2. A newcomer asks for assignment to an issue that has no
                    difficulty label.
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp18h0qdt6/plugins/labhub.py
+++ b/tmp/tmp18h0qdt6/plugins/labhub.py
@@ -289,8 +289,8 @@
                 if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(
                     user
                     )
-                      and not (self.TEAMS[self.GH_ORG_NAME +
-                                          ' developers'].is_member(
+                    and not (self.TEAMS[self.GH_ORG_NAME +
+                                        ' developers'].is_member(
                             user
                         ) or
                         self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(
```